src/Raven.Studio/typescript/models/database/index/configurationItem.ts

```diff
@@ -3,13 +3,16 @@ import jsonUtil = require("common/jsonUtil");
 import configuration = require("configuration");
 
 class configurationItem {
-    static readonly ConfigurationOptions = [
-        configuration.indexing.maxTimeForDocumentTransactionToRemainOpen,
-        configuration.indexing.minNumberOfMapAttemptsAfterWhichBatchWillBeCanceledIfRunningLowOnMemory,
-        configuration.indexing.mapTimeout,
-        configuration.indexing.mapTimeoutAfterEtagReached
+
+    static readonly ServerWideIndexingConfigurationOptions = [        
+        configuration.indexing.size,
```

Comment:
This list does not contain all ServerWide (only) configurations. Also the 'size' suggests that there is something clearly wrong with parsing those. I've created a separate issue to handle that. This is RegEx issue probably: https://issues.hibernatingrhinos.com/issue/RavenDB-14374

@Danielle9897 Can you fill the list with all the options that are parsed by the current implementation, then after RavenDB-14374 is done, we will update the list?

Reply:
Looking at _IndexingConfiguration.cs_ I see only 3 server-wide options :

* CleanupInterval
* GlobalScratchSpaceLimit
* MaxTimeToWaitAfterFlushAndSyncWhenExceedingScratchSpaceLimit

=> What are the other configurations that you mention ?

Reply:
I was missing 'GlobalScratchSpaceLimit' but I guess this is 'size' ?

Reply:
Yes
Taken from _configuration.ts:_

```
size: "Indexing.GlobalScratchSpaceLimitInMb",
cleanupInterval: "Indexing.CleanupIntervalInMin",
maxTimeToWaitAfterFlushAndSyncWhenExceedingScratchSpaceLimit: "Indexing.MaxTimeToWaitAfterFlushAndSyncWhenExceedingScratchSpaceLimit",
```